rust/kcl-lib/src/engine/mod.rs

```diff
@@ -253,6 +260,16 @@ pub trait EngineManager: std::fmt::Debug + Send + Sync + 'static {
             self.ensure_async_command_completed(id, Some(source_range)).await?;
         }
 
+        // Make sure we check for all async tasks as well.
+        // The reason why we ignore the error here is that, if a model fillets an edge
+        // we previously called something on, it might no longer exist. In which case,
+        // the artifact graph won't care either if its gone since you can't select it
+        // anymore anyways.
+        self.async_tasks().join_all().await.unwrap_or_default();
```

Comment:
Should we debug log or something here? Maybe with `logln!()`? Just seems like someone is going to want to know that we're ignoring an error here.

Reply:
yeah would logln show up in the cli thats my only fear

Reply:
or in the playwright tests, becasue i dont wanna fuck up console errrors there

Reply:
added

Reply:
I think you need to opt-in to log output using the environment variable `ZOO_LOG`.